src/content/docs/new-relic-one/use-new-relic-one/core-concepts/use-tags-help-organize-find-your-data.mdx

```diff
@@ -227,4 +242,73 @@ Learn about [how to use tags with dashboards](/docs/query-your-data/explore-quer
 
 ## Custom queries and charts [#query-tags]
 
-As explained in [Tag sources](#tag-sources), some [attributes](/docs/using-new-relic/welcome-new-relic/get-started/glossary#attribute) are available as tags. You can query attributes and create custom charts using [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language).
+Different features handle tags differently. Here are some details about how you can use [NRQL](/docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/) to query tag data.
+
+### Query APM tags [#query-apm-tags]
+
+You can query APM tags using NRQL. Minimum [agent versions](/docs/using-new-relic/cross-product-functions/install-configure/update-new-relic-agent): 
+
+* C: not available
+* Go: 2.3.0
+* Java: 4.9.0
+* .NET: 8.17
+* Node: v4.13.0
+* PHP: not available
+* Python: v4.10.0.112
+* Ruby: 6.1.0.352
+
+Here are some query examples: 
+
+<CollapserGroup>
+  <Collapser
+    id="apm-shard-distribution"
+    title="Throughput across shards"
+  >
+
+Given a service that is distributed across nine shards, each shard assigned as an `environment`, we can plot the transaction throughput on all shards using a query like: 
+
+```
+SELECT rate(count(apm.service.transaction.duration), 1 minute) 
+  FROM Metric WHERE appName LIKE MyApp (%' AND transactionType = 'Other' 
+  FACET tags.Environment TIMESERIES 
+```
+  </Collapser>
+
+  <Collapser
+    id="apm-team-tag"
+    title="Transactions per team"
+  >
+
+To see the number of transactions per service for a specific team, we might use a query like: 
```

Comment:
Not sure about using "might"

Reply:
i think this is okay just because it's an example. there are theoretically other queries one could use, in other words. another phrasing might be "you could".